src/reducers/iprove.js

```diff
@@ -92,7 +94,9 @@ const reducer = (state = initialState, action) => {
 
       case ADD_RELATIONS:
         const newRelations = newState.relations.concat(action.payload)
-        newState.relations = _.uniq(newRelations, false, _.iteratee('name'))
+        newState.relations = _.uniq(newRelations, function (r) {
+          return JSON.stringify({name:r.name, params: r.params})
+        });
```

Comment:
https://underscorejs.org/#uniq
It looks like you need 3 parameters, also better to do this (and remove semicolon)
`newState.relations = _.uniq(newRelations, ({ name, params }) => JSON.stringify({name, params }))`